.gitignore

```diff
@@ -131,4 +131,4 @@ iOSInjectionProject/
 /*.gcno
 **/xcshareddata/WorkspaceSettings.xcsettings
 
-# End of https://www.toptal.com/developers/gitignore/api/macos,xcode,swift
```

Comment:
Having another PR to change the .gitginore file will be safter, but it doesn't actually matter in this case.

Reply:
Will do it next time